examples/demo_portchannel.pp

```diff
@@ -0,0 +1,91 @@
+# Manifest to demo cisco_interface provider
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class ciscopuppet::demo_portchannel {
+
+  $port_hash_distribution = platform_get() ? {
+    /(n3k|n7k|n9k)/ => 'adaptive',
+    default => undef
+  }
+
+  $port_load_defer = platform_get() ? {
```

Comment:
Given the number of properties you are setting, let's keep these in alpha order as well.  Easier to read/reference.